code/game/area/areas/station.dm

```diff
@@ -1190,11 +1190,17 @@
 	icon_state = "checkpoint"
 
 /area/station/security/checkpoint/auxiliary
+	name = "\improper Auxiliary Security Checkpoint"
 	icon_state = "checkpoint_aux"
```

Comment:
It's great that you're giving it a name, but I've never liked this area definition in the first place because it is just incredibly non-verbose as to what an "Auxiliary" checkpoint is (or it's location on the station, unlike the Departures/Arrivals/Cargo/etc.) ones. It seems this area is only used on IceBoxStation, would you mind figuring out a better name for that specific location and adding that in code-side?

I'm only asking because your goal is to "clarify all of the different areas", but "auxiliary" is inherently an unclear term which just throws a wrench in the whole works.

Reply:
mark it draft you coward

Reply:
i didn't mark it draft because this is effectively a "want" and not a "need" that slightly transgresses past the initial point of this PR, and I only recalled it was a thing when this was turned over